tensorflow_io/core/python/experimental/elasticsearch_dataset_ops.py

```diff
@@ -106,7 +116,7 @@ def get_healthy_resource(self):
                 continue
         else:
             raise ConnectionError(
-                "No healthy node available for this index, check the cluster status and index"
+                "No healthy node available for this index, please check the cluster config"
```

Comment:
Consider printing out the healthcheck url/index

Reply:
addressed in the commit.